crates/e2e-tests/src/tests.rs

```diff
@@ -70,6 +70,50 @@ async fn test_wallet_api() -> Result<(), Box<dyn std::error::Error>> {
     let signature = signer.sign_hash_sync(&auth.signature_hash())?;
     let auth = auth.into_signed(signature);
 
+    let tx =
+        TransactionRequest::default().with_authorization_list(vec![auth]).with_to(signer.address());
+
+    let tx_hash: B256 = provider.client().request("odyssey_sendTransaction", vec![tx]).await?;
+
+    let receipt = PendingTransactionBuilder::new(provider.clone(), tx_hash).get_receipt().await?;
+
+    assert!(receipt.status());
+
+    assert!(!provider.get_code_at(signer.address()).await?.is_empty());
+
+    Ok(())
+}
+
+// Test legacy wallet API works
+#[tokio::test]
+async fn test_legacy_wallet_api() -> Result<(), Box<dyn std::error::Error>> {
```

Comment:
small nit: I would say best practise would be to leave the existing test as is and just add a new one. This way we can be sure 100% that the test wasn't changed and legacy path is still working. The new test makes sure that the new endpoint is working as well

Reply:
sure